### ISSUE TITLE
Converting collada will switch it from .dae to fbx for some reason

### ISSUE THREAD

**Issue by warriormaster12** [enziop/mixamo_converter#45]
When I want to add root motion to my collada formated animation I get an fbx output for some reason.

**Comment by enziop** [comment]
Hi, as the addon was built with Unreal Engine 4 as a target, it only exports to FBX for now.
I have not yet testet the root motion workflow for other formats and other engines yet, so i don't yet support the export.
If you know how you need the animations exported it should be easy to modify the addon yourself for your needs. In the file mixamoconv.py from line 511 - 519 the export happens. You can replace the command with the one you need in your local installation of the addon.
To find out what the export command for collada takes as arguments you can type the following line in the python console in blender and press shift + space
bpy.ops.wm.collada_export(
you can replace the fbx export line with the collada one set the proper settings you need.

I am planing to support more file formats and workflows for other engines in the future but, don't have the time to implement and test it at the moment. Collada and GLTF for Godot would be interesting for the future i think.

**Comment by enziop** [comment]
Just out of curiosity, where do you want to use the animations?

**Comment by warriormaster12** [comment]
I want to use root motion in godot. Funny thing is that I'm long time ue4 user but for some reason I'm more interested to try to make games in other engine. 

Btw what I did after exporting root motion enabled animation, I just imported it back to blender and exported as ".dae". It would be nice if your plugin would support that format in the future but I think that this method might work perfectly fine, MAYBE.
